models/models.topics.js

```diff
@@ -0,0 +1,14 @@
+const db = require("../db/connection.js");
+
+exports.selectTopics = () => {
+  const SQL = `SELECT * FROM topics;`;
+
+  return db
+    .query(SQL)
+    .then((data) => {
+      return data;
+    })
+    .catch((err) => {
```

Comment:
generally I'd say you don't need the catch block in the model - this is because if the model returns a failed the promise then the `.catch` in the controller will pick that up for you